bazel/cargo/wasmtime/Cargo.toml

```
@@ -10,8 +10,8 @@ path = "fake_lib.rs"
 env_logger = "0.8"
 anyhow = "1.0"
 once_cell = "1.3"
-wasmtime = {version = "0.35.2", default-features = false, features = ['cranelift']}
-wasmtime-c-api-macros = {git = "https://github.com/bytecodealliance/wasmtime", tag = "v0.35.2"}
+wasmtime = {version = "0.36.0", default-features = false, features = ['cranelift']}
```

```suggestion
wasmtime = {version = "0.36.0", default-features = false, features = ['cranelift', 'wasm-backtrace']}
```
You need to re-add `wasm-backtrace` feature, otherwise build fails.